src/main/scala/io/iohk/ethereum/consensus/atomixraft/Miner.scala

```diff
@@ -0,0 +1,162 @@
+package io.iohk.ethereum.consensus
+package atomixraft
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import akka.pattern.ask
+import akka.util.Timeout
+import io.iohk.ethereum.blockchain.sync.RegularSync
+import io.iohk.ethereum.consensus.atomixraft.Miner._
+import io.iohk.ethereum.domain.{Block, Blockchain}
+import io.iohk.ethereum.jsonrpc.EthService
+import io.iohk.ethereum.mining.{BlockGenerator, PendingBlock}
+import io.iohk.ethereum.nodebuilder.Node
+import io.iohk.ethereum.transactions.PendingTransactionsManager
+import io.iohk.ethereum.transactions.PendingTransactionsManager.PendingTransactionsResponse
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.{Failure, Success}
+
+class Miner(
```

Comment:
The word "miner" is specific to PoW. I believe "forger" is the most prevalent in PoS and everything else that doesn't depend on _work_.